doc/cephadm/services/monitoring.rst

```diff
@@ -344,10 +344,13 @@ Setting up Prometheus
 Setting Prometheus Retention Time
```

Comment:
```suggestion
Setting Prometheus Retention Time and Size
```
I think we ought to mention both in the section title

Reply:
fixed